applications/sdp/mspi/src/hrt/hrt.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _HRT_H__
+#define _HRT_H__
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#define SCLK_PIN 0
+#define D0_PIN	 1
+#define D1_PIN	 2
+#define D2_PIN	 3
+#define D3_PIN	 4
+#define CS_PIN	 5
```

Comment:
How can we correlate those defines with ones defined here:
https://github.com/nrfconnect/sdk-nrf/pull/18893/files#diff-648980e60dd56a6c243c32a1062ef806f0761ce8b043c5859065c794ffed4229R16-R29
As this is important for both cores how can we check its correctness?

Reply:
we can do it in this manner but only i the next PR after your changes will be merged:
#define NRFE_MSPI_SCK_PIN_NUMBER 1 /* Physical pins number on port 2 */
#define NRFE_MSPI_DQ0_PIN_NUMBER 2
#define NRFE_MSPI_DQ1_PIN_NUMBER 4
#define NRFE_MSPI_DQ2_PIN_NUMBER 3
#define NRFE_MSPI_DQ3_PIN_NUMBER 0
#define NRFE_MSPI_CS0_PIN_NUMBER 5

#define NRFE_MSPI_SCK_PIN_NUMBER_VIO 0 /* Logical VIO pins */
#define NRFE_MSPI_DQ0_PIN_NUMBER_VIO 1
#define NRFE_MSPI_DQ1_PIN_NUMBER_VIO 2
#define NRFE_MSPI_DQ2_PIN_NUMBER_VIO 3
#define NRFE_MSPI_DQ3_PIN_NUMBER_VIO 4
#define NRFE_MSPI_CS0_PIN_NUMBER_VIO 5

#define VIO(_PIN_) (_PIN_##_VIO)